armory/__init__.py

```diff
@@ -2,8 +2,7 @@
 Adversarial Robustness Evaluation Test Bed
 """
 from armory.logs import log
```

Comment:
This pull request is vastly easier to read with the Split Diff View under the gear icon up at top.